### PR TITLE
Clarify e-mail-address process

### PR DESCRIPTION
**Pull request by Ein-Tim**
**DE** 
image
Bildschirmfoto 2023-05-25 um 20 16 16

**EN** 
image
Bildschirmfoto 2023-05-25 um 20 16 36

Internal Tracking ID: [EXPOSUREAPP-15208](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-15208)

### CODE REVIEW

File: src/data/faq.json [hunk 1]
```diff
@@ -95,15 +95,15 @@
                                 "anchor": "ramp_down_mailbox",
                                 "active": false,
                                 "textblock": [
-                                    "Until May 31, 2023 there will be community support, in which incoming messages will be read and answered if applicable."
+                                    "The community support, which includes reading and answered if applicable to messages received at <a href='corona-warn-app.opensource...sap...com' class='email'>corona-warn-app.opensource...sap...com</a>, will continue until May 31, 2023. After that, the mailbox <a href='CoronaWarnApp...RKI...de' class='email'>CoronaWarnApp...RKI...de</a> will still be available, which is planned to be maintained until the end of 2023."
```
Comment (MikeMcC399) [inline src/data/faq.json:98]: "The community support, which includes reading and answered if applicable to messages received at"
=> 
"The community support, which includes reading and answering messages (if applicable) received by"

File: src/data/faq.json [hunk 2]
```diff
@@ -95,15 +95,15 @@
...
                                 ]
                             },
                             {
                                 "title": "What happens with my personal data in the emails?",
                                 "anchor": "ramp_down_mailbox_data",
                                 "active": false,
                                 "textblock": [
-                                    "An automatic deletion is set up for the CWA mailbox: All emails that are now in our CWA mailbox or still reach that mailbox will be deleted irreversibly without replacement, after 6 months at the latest, to ensure the protection of your data."
+                                    "An automatic deletion is set up for the CWA mailbox from SAP (<a href='corona-warn-app.opensource...sap...com' class='email'>corona-warn-app.opensource...sap...com</a>): All emails that are now in our CWA mailbox or still reach that mailbox will be deleted irreversibly without replacement, after 6 months at the latest, to ensure the protection of your data."
```
Comment (MikeMcC399) [inline src/data/faq.json:106]: "from SAP" => 
"of SAP"

"will be deleted irreversibly without replacement" =>
"will be completely deleted" (as translation of "vollständig gelöscht")

Comment (MikeMcC399) [reply src/data/faq.json:106]: "An automatic deletion" =>
"Automatic deletion"

File: src/data/faq.json (hunk 1 above)
Comment (Ein-Tim) [reply src/data/faq.json:98]:
```suggestion
                                    "The community support, which includes reading and answering messages (if applicable) received by <a href='corona-warn-app.opensource...sap...com' class='email'>corona-warn-app.opensource...sap...com</a>, will continue until May 31, 2023. After that, the mailbox <a href='CoronaWarnApp...RKI...de' class='email'>CoronaWarnApp...RKI...de</a> will still be available, which is planned to be maintained until the end of 2023."
```

File: src/data/faq.json (hunk 2 above)
Comment (Ein-Tim) [reply src/data/faq.json:106]:
```suggestion
                                    "Automatic deletion is set up for the CWA mailbox of SAP (<a href='corona-warn-app.opensource...sap...com' class='email'>corona-warn-app.opensource...sap...com</a>): All emails that are now in our CWA mailbox or still reach that mailbox will be completely deleted, after 6 months at the latest, to ensure the protection of your data."
```

File: src/data/faq.json (hunk 1 above)
Comment (Ein-Tim) [reply src/data/faq.json]: Fixed with [`86a47bc` (#3511)](https://github.com/corona-warn-app/cwa-website/pull/3511/commits/86a47bc365f962c5cff3dd5fb157bdb69ed1d02d).

**Review by dsarkar — APPROVED** [review]

**Comment by dsarkar** [comment]
@Ein-Tim I think we can merge this now, agreed?

**Comment by Ein-Tim** [comment]
@dsarkar Yes, please go ahead!

**Pull request merged**